.pre-commit-config.yaml

```diff
@@ -73,9 +69,7 @@ repos:
     rev: v2.5.0
     hooks:
       - id: prettier
-        files: ^js/.*$
-      - id: prettier
-        types: [yaml]
+        exclude: .*\.md$
```

Comment:
Just out of curiosity, why exclude Markdown files?

Reply:
Prettier has a very specific Markdown style guide that makes a lot of changes to our Markdown files, which would also ripple into other repos. We can enable it after a discussion about the scope of the changes.

Reply:
See #333.